tests/system/providers/google/marketing_platform/example_campaign_manager.py

```diff
@@ -103,64 +126,135 @@
     "kind": "dfareporting#conversion",
     "floodlightActivityId": FLOODLIGHT_ACTIVITY_ID,
     "floodlightConfigurationId": FLOODLIGHT_CONFIGURATION_ID,
-    "mobileDeviceId": DEVICE_ID,
+    "dclid": DCLID,
     "ordinal": "0",
     "quantity": 42,
     "value": 123.4,
 }
 
+
+log = logging.getLogger(__name__)
+
+
+def get_secret(secret_id: str) -> str:
+    hook = GoogleCloudSecretManagerHook()
+    if hook.secret_exists(secret_id=secret_id):
+        return hook.access_secret(secret_id=secret_id).payload.data.decode().strip()
+    raise NotFound("The secret '%s' not found", secret_id)
+
+
 with DAG(
     DAG_ID,
     schedule="@once",  # Override to match your needs,
     start_date=datetime(2021, 1, 1),
     catchup=False,
     tags=["example", "campaign"],
 ) as dag:
+
+    @task
+    def create_connection(connection_id: str) -> None:
+        connection = Connection(
+            conn_id=connection_id,
+            conn_type="google_cloud_platform",
+        )
+        conn_extra_json = json.dumps(
+            {
+                "scope": "https://www.googleapis.com/auth/cloud-platform,"
+                "https://www.googleapis.com/auth/ddmconversions,"
+                "https://www.googleapis.com/auth/dfareporting",
+                "impersonation_chain": "305048245045-compute@developer.gserviceaccount.com",
+                # "impersonation_chain": "service_account_with_campaign_manager_permissions",
```

Comment:
should we need this one line?

Reply:
We definitely need these scopes.
The comment is here intentionally, because this system test fails if you run it locally with your own service account. However, if you uncomment this line and place a default composer's service account from any project where our configuration script was executed, then it will work even locally. I left only a brief explanation at the top of the test, because I suppose that specific details shouldn't be exposed in open source.

Reply:
But only now I see that I left something that shouldn't be here. Updating the PR.

Reply:
Done. PTAL

Reply:
Maybe then we can make it as a variable? Because this version of the test will be uploaded from scratch every time we update the version of google-provider in our repo for system tests. And i don't think it is a good idea to go and fix this test every time. And the same should be then done for Freddy

Reply:
Done.